README.rst

```diff
@@ -1,297 +1,339 @@
 KNoT Network of Things - Gateway
-=============================================================================
-This is the gateway software module for the KNoT platform.
-It is based on buildroot 2014.11 version for RaspberryPi hardware.
-
-Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>  
-Copyright (C) 2006-2014 by the Buildroot developers <buildroot@uclibc.org>  
-Copyright (C) 2014 by the Buildroot developers <buildroot@buildroot.org>  
-Copyright (C) 2016 by the C.E.S.A.R KNoT developers <knot-devel@cesar.org.br>
```

Comment:
Can/should we remove this Copyright notices?

Reply:
On the mainline of Buildroot, this disclaimer does not exist.
The same disclaimer can be found on the Makefile and the COPYING file has the all license stuff.
So, in my opinion, we can remove this disclaimer. What do you think? Maybe we can leave the disclaimer because of CESAR Copyright.

Reply:
I agree that since BR removed from their README, it's OK. But need confirmation on the CESAR.

Reply:
I talked with Tiago Barros and he finds OK without this disclaimer.